lib/Bridge/TolerantParser/ResolveAliasSuggestionsTrait.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Phpactor\Completion\Bridge\TolerantParser;
+
+use Microsoft\PhpParser\Node;
+use Microsoft\PhpParser\ResolvedName;
+use Phpactor\Completion\Core\Suggestion;
+
+trait ResolveAliasSuggestionsTrait
```

Comment:
would it make sense to inject this rather than using a trait?

Reply:
That's why I tried to explain in the commit message.
I'm not sure it make sense to have a service for that since it simply provide methods to use.

But with your previous comment about the decorator I think it fixes the question.
If we can't transform it into a decorator then I might have a preference for a simple helper class with static methods